noxfile.py

```diff
@@ -17,20 +19,20 @@
 
 # In CI, use Python interpreter provided by GitHub Actions
 if CI:
-    nox.options.force_venv_backend = "none"
+    PYTHON_VERSIONS = [sys.executable]  # we don't disable venv, since we use it with pdm anyhow
```

Comment:
pdm sadly has no "install to system mode" AFAIK, but it is not big deal I think

on Dockerfile they use their own hacky way https://pdm.fming.dev/latest/usage/advanced/#use-pdm-in-a-multi-stage-dockerfile